interface-definitions/qos.xml.in

```diff
@@ -85,78 +85,67 @@
             <children>
               #include <include/generic-description.xml.i>
               #include <include/qos/bandwidth.xml.i>
-              <node name="flow-isolation">
```

Comment:
Moving this from a node -> leafNode and thus loosing the ability to select multiple flow-isolation options should suffice for a migration script? Or am I missinterpreting the change?

Reply:
done.

```
vyos@vyos:~$ cat /config/vyos-migrate.log 
List of applied migration modules:
applying /opt/vyatta/etc/config-migrate/migrate/qos/2-to-3

vyos@vyos:~$ diff /config/config.boot /config/config.boot.20241111-062019.pre-migration 
52,53c52,56
<             flow-isolation "src-host"
<             flow-isolation-nat
---
>             flow-isolation {
>                 dst-host
>                 nat
>                 src-host
>             }
```